llvm-bazel/llvm-project-overlay/llvm/BUILD

```diff
@@ -209,10 +209,13 @@ cc_library(
     ],
     copts = llvm_copts,
     includes = ["include"],
-    linkopts = [
-        "-pthread",
-        "-ldl",
-    ],
+    linkopts = select({
```

Comment:
Should we move the common selects that are used multiple times into a starlark macro?

Reply:
I commented on this in the description, but I think it's worth waiting to see how other platform configs shake out before refactoring this. It's not clear to me whether the current patterns (windows vs not-windows) will hold when we add additional configurations.